aspnetcore/data/ef-rp/complex-data-model.md

```diff
@@ -571,10 +571,17 @@ The ALTER TABLE statement conflicted with the FOREIGN KEY constraint "FK_dbo.Cou
 database "ContosoUniversity", table "dbo.Department", column 'DepartmentID'.
 ```
 
-When migrations are run with existing data, there may be FK constraints that are not satisfied with the exiting data. For this tutorial, a new DB is created, so there are no FK constraint violations. See
-[Fixing foreign key constraints with legacy data](#fk) for instructions on how to fix the FK violations on the current DB.
+When migrations are run with existing data, there may be FK constraints that are not satisfied with the existing data. For this tutorial, a new DB is created, so there are no FK constraint violations. See [Apply the migration to the existing database](#applyexisting) for instructions on how to fix the FK violations on the current DB.
```

Comment:
This explanation is given at the start of the "Apply migration to existing database" section, and in the immediately following section you explain the options, so I think we can remove it from here.